src/main/java/com/sap/sgs/phosphor/fosstars/data/github/UsesOwaspDependencyCheck.java

```diff
@@ -40,82 +66,250 @@ public UsesOwaspDependencyCheck(GitHubDataFetcher fetcher) {
   }
 
   @Override
-  protected Feature supportedFeature() {
-    return USES_OWASP_DEPENDENCY_CHECK;
+  protected Set<Feature> supportedFeatures() {
+    return setOf(
+        OWASP_DEPENDENCY_CHECK_USAGE, 
+        OWASP_DEPENDENCY_CHECK_FAIL_CVSS_THRESHOLD);
   }
 
   @Override
-  protected Value<Boolean> fetchValueFor(GitHubProject project) throws IOException {
+  protected ValueSet fetchValuesFor(GitHubProject project) throws IOException {
     logger.info("Figuring out if the project uses OWASP Dependency Check ...");
     LocalRepository repository = GitHubDataFetcher.localRepositoryFor(project);
-    boolean answer = checkMaven(repository) || checkGradle(repository);
-    return USES_OWASP_DEPENDENCY_CHECK.value(answer);
+
+    ValueSet values = new ValueHashSet();
+
+    checkMaven(repository, values);
+    if (found(values)) {
```

Comment:
I think we had discussed this?

If no values are found in Maven, would it still set default values? and `found()` would be `true`?

Reply:
When we discussed it last time, there were boolean features. Therefore the merge logic was quite simple. Here we have more complex features and the merge logic is not that straightforward. Since there are not too many projects that use both Maven and Gradle in the same time (have you seen any?), I would suggest to check Maven first. If the tools is not found, then check Gradle.

Currently there is definitely a bug in the `found()` method. I'll fix it by checking if the usage is not `NOT_USED`.